conda_forge_tick/migrators/nvtools.py

```diff
@@ -0,0 +1,244 @@
+import copy
+import logging
+import os.path
+from typing import Any
+
+import conda_forge_tick.migrators
+from conda_forge_tick.contexts import ClonedFeedstockContext
+from conda_forge_tick.migrators_types import AttrsTypedDict, MigrationUidTypedDict
+from conda_forge_tick.utils import (
+    get_bot_run_url,
+    yaml_safe_dump,
+    yaml_safe_load,
+)
+
+
+def _file_contains(filename, string):
+    with open(filename) as f:
+        return string in f.read()
+
+
+def _insert_requirements_build(meta):
+    insert_before: bool = True
```

Comment:
Think we can add this after compilers and like (assuming I'm understanding the intent of this parameter correctly)

```suggestion
    insert_before: bool = False
```

Reply:
This is not a function parameter. It is a variable which holds the state of "whether we are currently inserting lines into the variable named 'before'".

We're not going to do that because trying to locate the compiler packages and insert the new dependency after them is technically challenging and not necessary for a working migration. Adding the complexity of inserting after compiler packages instead of just inserting into requirements.build makes additional assumptions including 1. that the package has compiler packages listed in the top-level requirements.build section 2. the compiler packages are already sorted to the top of the requirements.build section